### PR TITLE
Return only active exams when asked for a list of proctored exams.

### PR DESCRIPTION
**Pull request by afzaledx**

### CODE REVIEW

**Comment by afzaledx** [comment]
@chrisndodge, @muhhshoaib

**Comment by chrisndodge** [comment]
@afzaledx we now have a merge conflict. I'll resolve and force push.

**Comment by chrisndodge** [comment]
@muhhshoaib can you do an initial review?

File: edx_proctoring/api.py [hunk 1]
```diff
@@ -811,7 +811,7 @@ def get_all_exams_for_course(course_id):
     ..
     ]
     """
-    exams = ProctoredExam.get_all_exams_for_course(course_id)
+    exams = ProctoredExam.get_all_exams_for_course(course_id, active_only)
```
Comment (muhhshoaib) [inline edx_proctoring/api.py]: I think we have to pass `active_only` as a keyword argument like `active_only= active_only`

File: edx_proctoring/api.py [hunk 2]
```diff
@@ -787,7 +787,7 @@ def remove_exam_attempt(attempt_id):
         )
 
 
-def get_all_exams_for_course(course_id):
+def get_all_exams_for_course(course_id, active_only=False):
```
Comment (muhhshoaib) [inline edx_proctoring/api.py]: We are already passing keyword argument in the models method. I think we can just pass simple parameter here. We are following the same structure elsewhere.

**Comment by muhhshoaib** [comment]
added some comments. Otherwise looks good to me :+1:

File: edx_proctoring/api.py (hunk 2 above)
Comment (afzaledx) [inline edx_proctoring/api.py]: active_only seems more readable than is_active.

File: edx_proctoring/api.py (hunk 1 above)
Comment (afzaledx) [inline edx_proctoring/api.py]: Made the change.

**Comment by chrisndodge** [comment]
@afzaledx I looked and there's no corresponding PR in edx-platform right? If so, please squash commits and merge.

**Comment by chrisndodge** [comment]
@afzaledx can you rebase and resolve conflicts. Looking at it, I'm not 100% how to resolve it myself. It's not too bad of a merge conflict. Then squash and merge. Thanks.

**Pull request merged**